src/components/TodoList/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import TodosCounter from '../TodosCounter';
+import TodoCreator from '../TodoCreator';
+
+function TodoList() {
+    const [newTodo, setNewTodo] = useState('')
+    const [todos, setTodos] = useState([])
+    useEffect(() => {
+        const initialTodos = [
+            'hacer ejercicio',
+            'tender la cama',
+            'desayunar',
+            'estudiar',
+            'hacer postwork'
+        ]
+        setTodos(initialTodos)
+    }, [])
+
+    function onUpdateTodo(value) {
+        setNewTodo(value)
+    }
+    function handleAddTodo() {
+        // Duplicar todos para evitar mutación con spread operator 
+        const todosCopy = [...todos]
+        todosCopy.push(newTodo)
+        setTodos(todosCopy)
+        setNewTodo('')
+    }
+    function handleRemoveTodo(index) {
+        // Confirm delete from user
+        let userConfirmed = window.confirm('Estás seguro de querer eliminar ésta tarea?')
+        if(!userConfirmed) {
+            return
+        }
+        const todosFiltered = todos.filter((_todo, i) => i !== index);
```

Comment:
Use better names instead of _todo and i, in general you want to give super easy to read context to whoever is reading the code